picturesafe-search-spring-boot-autoconfigure/src/test/java/de/picturesafe/search/autoconfigure/PicturesafeSearchAutoConfigurationTest.java

```diff
@@ -78,6 +91,16 @@ public void queryConfigurationIsNotNull() {
         assertThat(queryConfiguration).isNotNull();
     }
 
+    @Test
+    public void queryFactoriesIsNotNull() {
+        assertThat(queryFactories).isNotNull();
+    }
+
+    @Test
+    public void filterFactoriesIsNotNull() {
+        assertThat(filterFactories).isNotNull();
+    }
+
```

Comment:
Please check also that the lists are not empty. I had an empty list in the problematic configuration's bean context.

Reply:
Changed to isNotEmpty